adr/ADR-15.md

```diff
@@ -177,6 +177,20 @@ Why not append those new tokens at the end? This is to simplify the export/impor
 
 The account hash is not used by the client at this time, only used for routing, same for the last (12th) token.
 
+### Ack Handling
+
+An ack is accomplished by publishing a message using the JSAck as a subject 
+with the payload consisting of the bytes of the Ack Type
+
+Clients must support 3 terminal Ack Types, `+ACK`, `-NAK` and `+TERM` and the progress Ack Type `+WPI`.
+
+Once a terminal Ack Type has been published for a message, the client should ignore all other 
+user requests to send any type of ack. 
+
+If the consumer is configured with Ack Policy of `none`, the client should ignore all requests to send any type of ack.
```

Comment:
isn't this a noop in the server

Reply:
Likely, but in the same spirit that after a ack has been published, the library should ignore all other acks, then it would make sense to suppress acks for ack policy of "none". Although in that case, it would actually probably make more sense to return an error, because really the user should know that ack'ing a message does not make sense with ack none: simply ignoring gives a false sense that the user is in control of when the message is considered fully processed (or otherwise redelivered).